cli/src/lib.rs

```diff
@@ -333,6 +333,8 @@ where
         Ok(false)
     }
 
+    // Which raises the question: does it make sense to enable `nursery` lints?
```

Comment:
should we write a TODO here or remove the comment? Personally, I wouldn't enable nursery lints

Reply:
Created issue.